app/services/stories_api/v3/endpoints/story.rb

```diff
@@ -35,7 +36,12 @@ def get
         end
 
         def patch
-          story = user.user_sets.custom_find(params[:id])
+          story = if RecordSchema.roles[@user.role.to_sym].try(:admin)
+                    ::SupplejackApi::UserSet.custom_find(params[:id])
```

Comment:
There is no Story module, model or class.  The story interface is just an endpoint.